### PR TITLE
https://issues.redhat.com/browse/ACM-13326 move rest of ki

### PR DESCRIPTION
**Pull request by oafischer**
Rest of known issues to be moved after confirming with devs

### CODE REVIEW

File: release_notes/known_issues_observability.adoc [hunk 1]
```diff
@@ -254,3 +254,8 @@ Custom Observatorium API and Alertmanager URLs only support intermediate compone
 
 Metrics from an {ocp-short} 3.11 managed cluster (deprecated) are not displayed from the hub cluster because the metrics are not collected from your {acm-short} hub cluster.
 
+[#node-information-from-the-managed-cluster-cannot-be-viewed-in-search]
+=== Node information from the managed cluster cannot be viewed in search
+//2.0:4598
+
```
Comment (swopebe) [inline release_notes/known_issues_observability.adoc:254]: This one makes sense for this file, but we should make it clearer in the title that the issue is in Search:

_Search does not display_....
I will take this example to the team so we can better look at known issues and place them in the proper spot.

File: release_notes/known_issues_console.adoc [hunk 2]
```diff
@@ -94,3 +94,11 @@ To work around the issue, zoom out by using your browser zoom controls, increase
 == _EditApplicationSet_ expand feature repeats
 
 When you add multiple label expressions or attempt to enter your cluster selector for your `ApplicationSet`, you might receive the following message repeatedly,  "Expand to enter expression". You can enter your cluster selection despite this issue.
+
+[#ca-certificate-hub-restore]
+=== Restoring the connection of a managed cluster with custom CA certificates to its restored hub cluster might fail
```
Comment (swopebe) [inline release_notes/known_issues_console.adoc]: What is the reasoning behind placing this in the console?

File: release_notes/known_issues_application.adoc [hunk 3]
```diff
@@ -454,3 +454,54 @@ The following limitations occur if the hub cluster is not able to reach or commu
 Error querying resource logs:
 Service unavailable
 ----
+
```
Comment (swopebe) [inline release_notes/known_issues_application.adoc:345]: Application?

**Review by swopebe — CHANGES_REQUESTED** [review]
Thanks for tacking this. Two I am questioning the new placement.

File: release_notes/known_issues_console.adoc (hunk 2 above)
Comment (oafischer) [reply release_notes/known_issues_console.adoc]: I meant to place it in continuity, let me move it real quick

File: release_notes/known_issues_observability.adoc (hunk 1 above)
Comment (oafischer) [reply release_notes/known_issues_observability.adoc:254]: I'm updating the anchor too now that you mention it, way too long

File: release_notes/known_issues_application.adoc (hunk 3 above)
Comment (oafischer) [reply release_notes/known_issues_application.adoc:345]: Not sure where else to place it, what do you recommend?

**Pull request closed without merging**